### DISCUSSION TITLE
Bizonylatok tárolása az e-pénztárgépen felhőalapú szoftver esetén

### DISCUSSION THREAD

**Discussion by stajerg** [nav-gov-hu/eRECEIPT#28] [Kérdések és válaszok]
Az 1.0 verziószámú dokumentáció 9. fejezete szerint (53. oldal) : 

> Az adatokat az e-pénztárgépnek legalább naponta külön, adószámmal és dátummal megjelölt állományban kell tárolnia. Egy napon belül több állomány is létrehozható, ebben az esetben az állományokat napon belüli sorszámmal is el kell látni. Kétvállalkozós e-pénztárgépen az egyes adózókhoz tartozó adatok külön állományban kell szerepeljenek. Az egyes kérések az állományon belül időrendi sorrendben kell, hogy szerepeljenek.

Ez vonatkozik a felhőalapú szoftverekre is? Ott általában egy adatbázis szokott lenni, és nem állományokban vannak tárolva az egyes üzenetek.

**Comment by renced42** [comment]
Kedves @stajerg 

A felhő alapú ePénztárgép jelen pillanatban egy készüléken futó szoftver, amely a NAV rendszeréhez csatlakozik.
Lényeges, hogy minden logika a szoftverben fut az eszközön, nem pedig egy távoli szerveren.

**Reply by westito** [reply]
A készüléken is adatbázisban szokás tárolni az adatokat és nem külön fájlokban. Nem elvárható hogy a szoftver fájlokból olvassa és keresse vissza ezeket. Úgy tudom elképzelni a megfelelést, hogy az adatbázis mellett külön fájlokban is eltárolásra kerülnek az adatok.

**Comment by renced42** [comment]
Kedves @westito @stajerg 

Az, hogy ki hogy tárolja az adatokat az teljesen a megvalósítástól függ. A napló fájlokat lehet DB-ben és filerendszerben is tárolni.
